src/gobble.py

```diff
@@ -19,26 +21,72 @@
 
 API_KEY = CONFIG["mbta"]["v3_api_key"]
 HEADERS = {"X-API-KEY": API_KEY, "Accept": "text/event-stream"}
-URL = f'https://api-v3.mbta.com/vehicles?filter[route]={",".join(ALL_ROUTES)}'
 
 
 def main():
-    current_stop_state: Dict = disk.read_state()
-
     # Download the gtfs bundle before we proceed so we don't have to wait
     logger.info("Downloading GTFS bundle if necessary...")
     gtfs_service_date = util.service_date(datetime.now(util.EASTERN_TIME))
     scheduled_trips, scheduled_stop_times, stops = gtfs.read_gtfs(gtfs_service_date)
 
-    logger.info(f"Connecting to {URL}...")
-    client = sseclient.SSEClient(requests.get(URL, headers=HEADERS, stream=True))
+    rapid_url = f'https://api-v3.mbta.com/vehicles?filter[route]={",".join(ROUTES_RAPID)}'
```

Comment:
this looks pretty sick, but im wondering if this is quite how we want to split up our lines across threads--we have many busses with frequent stops, fewer/less frequent rapid transit, and even fewer CR events. maybe we can split half the busses on one thread, half on another, and combine CR+rapid on a third thread?

(i realize that the gps ping frequency is about the same for all the vehicles, but the total number of vehicles and density of actionable events will be higher on the bus lines)

Reply:
chunks of max 10 sounds good to me! if we still see issues, we can reduce that or be a little more mindful of how we balance the more frequent busses across the threads.